Lib/test/test_import/__init__.py

```diff
@@ -812,9 +813,37 @@ def test_frozen_module_from_import_error(self):
             from os import this_will_never_exist
         self.assertRegex(
             str(cm.exception),
-            r"cannot import name 'this_will_never_exist' from 'os' \(.*Lib[\\/]os\.py\)"
+            f"cannot import name 'this_will_never_exist' from 'os' \\({re.escape(os.__file__)}\\)"
```

Comment:
It may be simpler to call `re.escape()` for the whole string.
```suggestion
            re.escape(f"cannot import name 'this_will_never_exist' from 'os' ({os.__file__})")
```
Or do not use `assertRegex`.